src/interpreter/repl.rs

```diff
@@ -20,50 +23,90 @@ impl<'a> Default for FenderRepl<'a> {
     }
 }
 
+fn display_result(result: Result<FenderReference, Box<dyn Error>>) {
+    match result {
+        Ok(FenderReference::FRaw(FenderValue::Null)) => (),
+        Ok(val) => println!("{}", val.to_string()),
+        Err(err) => eprintln!("{err:#}"),
+    }
+}
+
 impl<'a> FenderRepl<'a> {
     pub fn new() -> FenderRepl<'a> {
         let mut engine = ExecutionEngine::new_default();
-        let scope = LexicalScope {
-            labels: HashMap::new().into(),
-            args: ArgCount::Fixed(0),
-            captures: vec![].into(),
-            variables: HashMap::new().into(),
-            parent: None,
-            return_target: engine.create_return_target(),
-            num_stack_vars: 0,
-        };
-        FenderRepl { engine, scope }
+        let scope = LexicalScope::new(ArgCount::Fixed(0), engine.create_return_target());
+        FenderRepl {
+            engine,
+            scope,
+            editor: DefaultEditor::new().expect("Prompt created"),
```

Comment:
this sounds like the error msg is saying what happened is actually what failed to happen